projects/soft_999_eddy_kim_fw_103/src/main.c

```diff
@@ -0,0 +1,49 @@
+#include "adc.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+
+static void callback(const GpioAddress *address, void *context) {
```

Comment:
This is a private function, so by our conventions it should be prefixed with `prv_`.